pkg/pillar/cmd/baseosmgr/baseosmgr.go

```diff
@@ -371,7 +293,7 @@ func initializeZedagentHandles(ps *pubsub.PubSub, ctx *baseOsMgrContext) {
 }
 
 func initializeVolumemgrHandles(ps *pubsub.PubSub, ctx *baseOsMgrContext) {
-	// Look for BaseOs OldVolumeStatus from volumemgr
+	// Look for BaseOs VolumeStatus from volumemgr
```

Comment:
Wrong comment. Revert change

Reply:
How was it resolved? I don't understand why this PR would touch anything related to Volume*